myhpom/tests/test_context_processors.py

```diff
@@ -0,0 +1,12 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+
+class ContactEmailProcessorTestCase(TestCase):
+    def test_email_is_available(self):
+        for example_email in ('foo@example.com', 'bar@example.com', ):
+            with self.settings(CONTACT_EMAIL=example_email):
+                # arbitrary view: they should all have it!
+                response = self.client.get(reverse('myhpom:home'))
```

Comment:
I'd feel a little more secure with the test if it looped all the GET-able views – or at least verified that the view itself doesn't add 'example_email' to the context.